lms/templates/navbar.html

```diff
@@ -37,11 +37,31 @@
             </div>
           </div>
         </li>
-        <li class="nav-item">
-          <a href="/send" class="nav-link">
-            <i class="fa fa-upload" aria-hidden="true"></i>
-            {{ _('העלאת תרגילים') }}
+        <li class="nav-item dropdown">
+          <a id="courses-icon" href="#" class="nav-link" aria-expanded="false" aria-haspopup="true" aria-label="Courses" role="button" data-toggle="dropdown">
+            <i class="fa fa-graduation-cap" aria-hidden="true"></i>
+            {% if not current_user.last_course_viewed %}
+              {{ _('רשימת הקורסים') }}
+            {% else %}
+              {{ current_user.last_course_viewed.name | e }}
+            {% endif %}
           </a>
+          <div id="courses-list" aria-labelledby="navbarDropdown" class="dropdown-menu 
```

Comment:
Maybe we shall show it only if the user has registered to more then one course?

Reply:
Can we do it in a new feature? Because there is the complicated situation whether a user is registered for a course or not.